fili-core/src/main/java/com/yahoo/bard/webservice/data/DruidQueryBuilder.java

```diff
@@ -367,8 +370,9 @@ protected TimeSeriesQuery buildTimeSeriesQuery(
 
         LOG.trace("Building a single pass druid timeseries query");
 
+        // TODO FIXME hardcoding to concrete for now
         // The data source is the table directly, since there is no nested query below us
-        DataSource dataSource = new TableDataSource(table);
+        DataSource dataSource = new TableDataSource((ConcretePhysicalTable) table);
```

Comment:
Alright, I'm putting in a slightly more resilient variation.  However, we should still tweak this in some way.